jsconcat.php

```diff
@@ -89,6 +89,13 @@ function do_items( $handles = false, $group = false ) {
 			else
 				$js_url_parsed['path'] = substr( $js_realpath, strlen( ABSPATH ) - 1 );
 
+			// Check for inline translations
+			$translations = $this->print_translations( $handle, false );
+			if ( $translations ) {
+				$do_concat = false;
+				$translations = sprintf( "<script type='text/javascript'>\n%s\n</script>\n", $translations );
```

Comment:
This looks un-used? Looks like the `$before_handle` and `$after_handle` values below end up un-used as well.

Reply:
Good point.  What about something like this?

```
$translations = $this->print_translations( $handle, false );

// Check for scripts added from wp_add_inline_script()
$before_handle = $this->print_inline_script( $handle, 'before', false );
$after_handle = $this->print_inline_script( $handle, 'after', false );

if ( $before_handle || $after_handle || $translations ) {
	$do_concat = false;
}
```

Reply:
Rather than all the overhead of `print_translations`, I wonder if it's better to copy the check it has:

```
// Has js translations so we can't concat
if ( ! empty( $this->registered[ $handle ]->textdomain ) {
    $do_concat = false;
}
```

Reply:
Can probably do the same thing for `print_inline_script` and combine into a helper function for easier use:

```
protected function has_inline_content( $handle ) {
    $before_output = $this->get_data( $handle, 'before' );
    if ( ! empty( $before_output ) ) {
        return true;
    }

    $after_output = $this->get_data( $handle, 'after' );
    if ( ! empty( $after_output ) ) {
        return true;
    }

    // JavaScript translations
    $has_translations = ! empty( $this->registered[ $handle ]->textdomain );
    if ( $has_translations ) {
        return true;
    }

    return false;
}
```

Reply:
Thanks @mjangda!  Great suggestion.  I tested this on the site with the issue and have now updated this PR with your suggestion.